setup.cfg

```
@@ -97,13 +97,13 @@ all =
     fsspec[http]>=2022.8.2
     s3fs>=2022.8.2
 docs =
-    sphinx<4
+    sphinx
     sphinx-astropy>=1.6
     pytest>=7.0
     scipy>=1.3
     matplotlib>=3.1,!=3.4.0,!=3.5.2
     sphinx-changelog>=1.2.0
-    Jinja2>=3.0,<3.1
+    Jinja2
```

Just have to remove the max pin.
```suggestion
    Jinja2>=3.0
```

---

I think Jinja2 can be removed, it's a dependency of Sphinx and was pinned only to get a version compatible with Sphinx<4.

---

We do explicitly use it in our conf.py file so I think it should stay.

---

sphinx's `pyproject.toml` has an explicit "Jinja2>=3.0" -- my tendency would be to assume it remains a requirement of `sphinx` and let them worry about version control -- one cannot run `conf.py` without `sphinx` anyway. @cadair - I'm not quite sure I follow why you feel it should stay.

---

Actually, let me just push @pllim's minimum requirement so we can get this in!